rpcs3/Emu/System.cpp

```diff
@@ -694,17 +694,13 @@ game_boot_result Emulator::GetElfPathFromDir(std::string& elf_path, const std::s
 
 game_boot_result Emulator::BootGame(const std::string& path, const std::string& title_id, bool direct, bool add_only, cfg_mode config_mode, const std::string& config_path)
 {
-	if (!fs::exists(path))
-	{
-		return game_boot_result::invalid_file_or_folder;
-	}
-
```

Comment:
the idea here is that you don't overwrite members if the boot file does not exist.
If you're gonna add weird logic like this, at least make sure it's gonna work before you call the boot function

Reply:
I've added a better fix for it.